scripts/typesFiles.js

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env node
+const fs = require("fs");
+// eslint-disable-next-line import/no-extraneous-dependencies
+const shell = require("shelljs");
+// eslint-disable-next-line import/no-extraneous-dependencies
+const reactDocs = require("react-docgen");
+
+const skipPackages = [
+  "Guard",
+  "Icon",
+  "Stylers",
+  "Tokens",
```

Comment:
how Tokens has a declaration file but is skip it here? does the `tsc` script create for us? if so, then can we differentiated which packages are skip that we can't process by now, and which are those which are skip because `tsc` process via the `tscofing`?

Reply:
Yes, `token.d.ts` was processed using `tsc` so it was added to the the skipPackages. Will look into a better way to differentiate skipPackages and files created via `tsc`. 👍